Kill "update location by sms"

esoergel opened the pull request:
Remove unused "update location by sms" feature.  Found some additional context here: https://github.com/dimagi/commcare-hq/pull/21856

@millerdev Can you validate that this looks like a reasonable change?  In particular, the #UPDATE keyword now seems vestigial, so I removed that too.

esoergel committed:
Remove unused "update location by sms" feature

Found some additional context here: https://github.com/dimagi/commcare-hq/pull/21856

esoergel committed:
Remove update keyword that no longer does anything

millerdev reviewed (commented):
Looks good to me. Test failures are real:
```
ERROR: nose.failure:Failure.runTest
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/vendor/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/vendor/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/mnt/commcare-hq-ro/custom/intrahealth/tests/test_fluffs.py", line 14, in <module>
    from custom.intrahealth.tests.test_utils import IntraHealthTestCase, TEST_DOMAIN
  File "/mnt/commcare-hq-ro/custom/intrahealth/tests/test_utils.py", line 10, in <module>
    from corehq.apps.sms.tests.update_location_keyword_test import create_mobile_worker
ImportError: No module named update_location_keyword_test
```

esoergel committed:
Use the create method

This utility doesn't do anything beyond this unless the last option is
provided

esoergel commented:
Wow, that is the strangest coupling of code... it doesn't even need the optional argument, so you can drop-in replace `CommCareUser.create`. Updated
